### PR TITLE
Size of binary tree added

### PR DESCRIPTION
**Pull request by shivammahajancse**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Yes)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(#330 )

### CODE REVIEW

**Comment by shivammahajancse** [comment]
Please review sir @HarshCasper @iamrajiv @ankitaggarwal23 @atarax665 @gat786

File: docs/data-structures/size-of-binary-tree.mdx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+print(size(root))
+     
+```
+</TabItem>
+</Tabs>
+
+## Sample Input and Output
+
+#### Input :
+
```
Comment (iamrajiv) [inline docs/data-structures/size-of-binary-tree.mdx:139]: Remove this line break.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@shivammahajancse Resolve the merge conflict.

**Comment by atarax665** [comment]
Resolve conflict @shivammahajancse

**Review by atarax665 — APPROVED** [review]

**Review by iamrajiv — APPROVED** [review]

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@shivammahajancse Resolve the merge conflict so that I can merge the PR.

**Comment by shivammahajancse** [comment]
Done sir @iamrajiv

**Pull request merged**